packages/react-storage/src/components/StorageBrowser/index.ts

```diff
@@ -7,3 +7,4 @@ export {
   StorageBrowserAuthAdapter,
 } from './adapters';
 export { CreateStorageBrowserInput, StorageBrowserType } from './types';
+export { DEFAULT_STORAGE_BROWSER_DISPLAY_TEXT } from './displayText';
```

Comment:
Need to export this so I can use it in the docs. For some reason it didn't like importing that file from a relative path

Reply:
This doesn't feel right IMO, if it we absolutely need it the export needs to be marked with an `@Internal` flag and provided through a non-public entry point

Reply:
Discussed offline, it does make sense to expose it, but change the casing